src/vsr.zig

```diff
@@ -1480,6 +1480,52 @@ pub const Checkpoint = struct {
         assert(constants.journal_slot_count > constants.lsm_batch_multiple);
         assert(constants.journal_slot_count % constants.lsm_batch_multiple == 0);
     }
+    /// Checkpoint identifier.
```

Comment:
Useless comment!

Would be good to explain _what_ it is (checksum of the CheckpointState in the superblock).